### ISSUE TITLE
Write a progress report for Jun/Jul/Aug/Sept

### ISSUE THREAD

**Issue by skmp** [reicast/reicast-emulator#1389]
It's been many many months, we need to summarize the work done, and document the release changes.

**Comment by skmp** [comment]
@luserx0 You said in the past you'd like to do this. Do you have the time/comitment to do it before oct 1st?

**Comment by luserx0** [comment]
@skmp Yes I can do it. The only problem is that I'm around for the past 1 month, so you'll have to be a little more specific on what the progress report should contain :)

**Comment by baka0815** [comment]
As we have some kind of report for the release of 8.1, I tried to write a progress for Sep and Oct myself (see https://github.com/baka0815/reicast.com/tree/progress_next). Is that the style we want to go?

Should I finish that up (rename the file to the correct date, correct the date in the post) and create a PR?
